packages/ui/src/facade/f-univer.ts

```diff
@@ -43,6 +60,11 @@ export interface IFUniverUIMixin {
      * @returns The component manager
      */
     getComponentManager(): ComponentManager;
+
+    /**
+     * Show a message.
+     */
```

Comment:
please add an example for using meesage options, 
![image](https://github.com/user-attachments/assets/799df3cd-2d09-4e1c-aa6f-5c933432b82c)

Reply:
This image seems inrelevant?